### DISCUSSION TITLE
有没有Actor substitution/Genre substitution已经生成好的列表？

### DISCUSSION THREAD

**Discussion by tyggg** [metatube-community/jellyfin-plugin-metatube#308] [Ideas]
没有的话我自己做一份

**Comment by Izumade** [comment]
Just finished my list.  48433 Actor/Actress substitution, 356 Genre substitution i used [javlibrary.com](https://www.javlibrary.com/en/) as source. Formatted ready to use just copy out auf the .txt file

Example; 安倍仁美=Abe Hitomi

[Metatube Actress Subsituitions 48433 A-Z 2024-06-08.txt](https://github.com/user-attachments/files/15745469/Metatube.Actress.Subsituitions.48433.A-Z.2024-06-08.txt)

[Metatube Genre Subsituitions 356 2024-06-08.txt](https://github.com/user-attachments/files/15745471/Metatube.Genre.Subsituitions.356.2024-06-08.txt)

**Comment by chu-shen** [comment]
source: JAVLibrary

[Metatube.Genre.Subsituitions.JP-ZH.20250329.txt](https://github.com/user-attachments/files/19521975/Metatube.Genre.Subsituitions.JP-ZH.20250329.txt)
